sqldb/migrations_test.go

```diff
@@ -570,3 +570,164 @@ func TestSchemaMigrationIdempotency(t *testing.T) {
 		}
 	})
 }
+
+// TestMigrationBug19RC1 tests a bug that was present in the migration code
+// at the v0.19.0-rc1 release.
+// The bug was fixed in: https://github.com/lightningnetwork/lnd/pull/9647
+// NOTE: this test may be removed once the funal version of 0.19.0 is released.
+func TestMigrationBug19RC1(t *testing.T) {
```

Comment:
we are not really testing a bug here but rather the bug will not happen, so maybe call it

`TestMigrationSucceedsAfterDirtyStateMigrationFailure19RC1`

Reply:
good point, done!